src/test/java/net/snowflake/client/jdbc/PreparedStatementIT.java

```diff
@@ -521,9 +560,13 @@ public void testClearParameters() throws SQLException
     prepStatement = connection.prepareStatement(insertSQL);
     bindOneParamSet(prepStatement, 1, 1.22222, (float) 1.2, "test", 12121212121L, (short) 12);
     prepStatement.clearParameters();
+
+    int parameterSize =
+        ((SnowflakePreparedStatementV1)prepStatement).getParameterBindings().size();
+    assertThat(parameterSize, is(0));
+
     bindOneParamSet(prepStatement, 3, 1.22, 1.2f, "hello", 12222L, (short) 1);
-    prepStatement.addBatch();
-    prepStatement.executeBatch();
+    prepStatement.executeUpdate();
```

Comment:
why this was changed? I believe the intention was clearParameters will not impact the following update.

Reply:
There is another test called PreparedStatementIT.testClearBatch() which cover the add batch case/clear batch case. This line is under test `testClearParameters()`. My original intention is to test whether parameters are cleared after `executeUpdate`, and it seems we don't. Not sure if we should do that.

Reply:
ok, I see testClearBatch covers it.